ciao-controller/controller_test.go

```diff
@@ -784,7 +770,12 @@ func testStartWorkload(t *testing.T, num int, fail bool, reason payloads.StartFa
 		t.Fatal(err)
 	}
 
-	client := newTestClient(0, ssntp.AGENT)
+	client, err := testutil.NewSsntpTestClientConnection("StartWorkload", ssntp.AGENT)
+	if err != nil {
+		t.Fatal(err)
+	}
+	// deliberately leak
+	//defer client.Ssntp.Close()
 
```

Comment:
Same comment as above, replace the comment "deliberately leak", with "Caller is responsible for closing connection".